charts/atlantis/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: v0.20.1
 description: A Helm chart for Atlantis https://www.runatlantis.io
 name: atlantis
-version: 4.1.2
+version: 4.2.0
```

Sorry, we already released 4.2.0, please update to 4.3.0

```suggestion
version: 4.3.0
```